src/components/SmallBoard.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="board column is-6">
+    <h2 class="board-name">
+      <slot></slot>
+    </h2>
+    <div
+    v-for="(article, index) in listitems.recent_articles"
+    :key="article.id"
+    class="post">
```

Comment:
I was just wondering, what's the difference between _article_ and _post_?

Reply:
Two are same, but it is just because backend returned as 'recent_articles'. I'll change it to 'post'